plugins/network-elements/nsx/src/main/java/org/apache/cloudstack/resource/NsxResource.java

```diff
@@ -428,6 +431,14 @@ private String getSegmentName(CreateNsxSegmentCommand cmd) {
          return segmentName + cmd.getVpcName() + "-" + cmd.getTierNetwork().getName();
     }
 
+    private String getDhcpRelayId(CreateNsxSegmentCommand cmd) {
+        String suffix = "-Relay";
+        if (isNull(cmd.getVpcName())) {
+            return cmd.getTierNetwork() + suffix;
+        }
+        return String.format("%s-%s%s", cmd.getVpcName(), cmd.getTierNetwork(), suffix);
```

Comment:
I think we should unify this as a helper method similar to `getSegmentName()` to be used  on the relay creation and deletion

Reply:
Since they were only referenced in the NsxResource class, I didn't create a utility, put rather created functions in the same class..